packages/library-base/src/commonMain/kotlin/io/realm/kotlin/dynamic/DynamicMutableRealm.kt

```diff
@@ -57,6 +58,27 @@ public interface DynamicMutableRealm : DynamicRealm {
      */
     public fun copyToRealm(obj: DynamicRealmObject, updatePolicy: UpdatePolicy = UpdatePolicy.ERROR): DynamicMutableRealmObject
 
+    /**
+     * Copy new objects into the realm or update existing objects without returning the managed
+     * objects afterwards
+     *
+     * If you do not want to use the returned object, this method is generally faster than using
+     * [copyToRealm] since it is possible to optimize memory allocations. For bulk inserting data,
+     * this method is thus preferred to using [copyToRealm].
+     *
+     * Otherwise the behaviour is similar to [copyToRealm].
+     *
+     * @param obj the object to create a copy from.
+     * @param updatePolicy update policy when importing objects. Asymmetric Realm objects
+     * only support [UpdatePolicy.ERROR]. If the wrong update policy is used an [IllegalArgumentException]
+     * is thrown.
+     * @throws IllegalArgumentException if the object graph of [obj] either contains an object
+     * with a primary key value that already exists and the update policy is [UpdatePolicy.ERROR],
+     * if the object graph contains an object from a previous version or if a property does not
+     * match the underlying schema.
+     */
+    public fun insert(obj: DynamicRealmObject, updatePolicy: UpdatePolicy = UpdatePolicy.ERROR)
```

Comment:
Yeah, I actually thought I removed this again. It is true, this PR don't need this.

Reply:
After thinking a bit more, this was moved to an extension function similar to `MutableRealm` and the interface was aligned with that. This means that it will only accept asymmetric objects for now and has been marked with an Experimental annotation.